content/rs/databases/memory-performance/memory-limit.md

```diff
@@ -16,64 +16,85 @@ aliases: [
+- [**modules**]({{<relref "/modules/_index.md">}}): using modules with your database consumes more memory.
+- [**database clustering**]({{<relref "/rs/databases/durability-ha/clustering.md">}}): enables you to spread your data into shards across multiple nodes.
+- [**database replication**]({{<relref "/rs/databases/durability-ha/replication.md">}}): enabling replication doubles memory consumption.
+
+Additional factors for Active-Active databases:
+
+- [**Active-Active replication**]({{<relref "/rs/databases/active-active/_index.md">}}): enabling Active-Active replication requires double the memory of regular replication, which can be up to two times (2x) the original data size per instance.
+- [**database replication backlog**]({{<relref "/rs/databases/active-active/manage#replication-backlog/">}}) for synchronization between shards. By default, this is set to 1% of the database size.
+- [**Active-Active replication backlog**]({{<relref "/rs/databases/active-active/manage.md">}}) for synchronization between clusters. By default, this is set to 1% of the database size.
+
+  It's also important to know Active-Active databases have a lower threshold for activating the eviction policy, because it requires propagation to all participating clusters. The eviction policy starts to evict keys when one of the Active-Active instances reaches 80% of its memory limit.
+
+Additional factors for Redis on Flash databases:
+
+- [**database persistence**]({{<relref "/rs/databases/configure/database-persistence.md">}}): Redis on Flash uses dual database persistence where both the primary and replica shards persist to disk. This may add some processor and network overhead, especially in cloud configurations with network attached storage.
+
+## What happens when Redis Enterprise Software is low on RAM?
 
-## Example 1
+Redis Enterprise Software manages node memory so that data is entirely in RAM (unless using Redis on Flash). If not enough RAM is available, RS prevents adding more data into the databases.
 
-You create a database and:
+Redis Enterprise Software protects the existing data and prevents the database from being able to store data into the shards.
 
-- Set the memory limit to 4 GB
-- Enable database replication in order to ensure high-availability
+You can configure the cluster to move the data to another node, or even discard it according to the [eviction policy]({{< relref "/rs/databases/memory-performance/eviction-policy.md" >}}) set on each database by the administrator.
 
-The cluster creates two shards: a primary and a replica. Each 
-shard has a maximum size of 2 GB. In this case, the maximum
-dataset size that you can store in the database is 2 GB.
+[Redis on Flash]({{< relref "/rs/databases/redis-on-flash/" >}})
+manages memory so that you can also use flash memory (SSD) to store data.
 
-## Example 2
+### If a node is low on RAM, RS follows this order of priority
```

Comment:
```suggestion
### Low-RAM action priority
If a node is low on RAM, Reds Enterprise Software follows this order of priority:
```
I don't know if this exact suggestion is perfect, but I think this heading might be better as a sentence.